_includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md

```diff
@@ -0,0 +1,385 @@
+        <action>index</action>
+        <pages>
+            <create-offer>
+                <label>Create Offer</label>
+                <title>Create Offer</title>
+                <bundle>product-offer-merchant-portal-gui</bundle>
+                <controller>product-list</controller>
+                <action>index</action>
+                <visible>0</visible>
+            </create-offer>
+        </pages>
+    </product-offer-merchant-portal-gui>
+    <sales-merchant-portal-gui>
+        <label>Orders</label>
+        <title>Orders</title>
+        <icon>orders</icon>
+        <bundle>sales-merchant-portal-gui</bundle>
+        <controller>orders</controller>
+        <action>index</action>
+    </sales-merchant-portal-gui>
+    <product-merchant-portal-gui>
+        <label>Products</label>
+        <title>Products</title>
+        <icon>offers</icon>
+        <bundle>product-merchant-portal-gui</bundle>
+        <controller>products</controller>
+        <action>index</action>
+    </product-merchant-portal-gui>
+    <security-merchant-portal-gui>
+        <label>Logout</label>
+        <title>Logout</title>
+        <icon>logout</icon>
+        <bundle>security-merchant-portal-gui</bundle>
+        <controller>logout</controller>
+        <action>index</action>
+    </security-merchant-portal-gui>
+</config>
+```
+</details>
+
+2. Build the navigation cache:
+
+```bash
+console navigation:build-cache
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the following applies:
+* The Merchant Portal navigation contains all configured items.
+* Clicking the configured navigation items opens respective pages.
+* In `\Pyz\Zed\ZedNavigation\ZedNavigationDependencyProvider`, `\Spryker\Zed\Acl\Communication\Plugin\Navigation\AclNavigationItemCollectionFilterPlugin` is enabled:
+
+```php
+<?php
+
+namespace Pyz\Zed\ZedNavigation;
+
+use Spryker\Zed\Acl\Communication\Plugin\Navigation\AclNavigationItemCollectionFilterPlugin;
+use Spryker\Zed\ZedNavigation\ZedNavigationDependencyProvider as SprykerZedNavigationDependencyProvider;
+
+class ZedNavigationDependencyProvider extends SprykerZedNavigationDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\ZedNavigationExtension\Dependency\Plugin\NavigationItemCollectionFilterPluginInterface>
+     */
+    protected function getNavigationItemCollectionFilterPlugins(): array
+    {
+        return [
+            new AclNavigationItemCollectionFilterPlugin(),
+        ];
+    }
+}
+```
+
+{% endinfo_block %}
+
+### 5) Separate login feature setup (security firewalls).
+
+It requires upgrading `spryker/smyfony:3.5.0` and applying some changes on the project. For details, see [Symfony 5 integration](/docs/dg/dev/upgrade-and-migrate/upgrade-to-symfony-5.html).
+
+1. Install the required modules using Composer:
+
+```bash
+composer remove spryker/auth spryker/auth-mail-connector spryker/auth-mail-connector-extension spryker/authentication-merchant-portal-gui
+```
+
+```bash
```

Comment:
Instructions missing

Reply:
Instructions are higher for both of the console commands: `Install the required modules using Composer:`